.github/workflows/benchmark.yml

```diff
@@ -11,9 +11,23 @@ on:
 jobs:
   run_benchmarks:
     runs-on: 'ubuntu-22.04'
+    strategy:
+      max-parallel: 1
+      fail-fast: false
+      matrix:
+        include:
```

Comment:
I think we'd just want one of their approaches rather than all three discussed in the article?